### PR TITLE
Next(content): Mock Overview content

### PR DESCRIPTION
**Pull request by rowa-audil**
Adding some basic styling to the next app.
Renames the BaPageService to DsPageService.

Fixes: #1210

### CODE REVIEW

File: nx.json [hunk 1]
```diff
@@ -361,7 +361,12 @@
     },
     "next": {
       "tags": ["scope:design-system", "type:app"],
-      "implicitDependencies": ["fluid-button"]
+      "implicitDependencies": [
+        "fluid-button",
+        "shared-data-access-strapi",
+        "shared-design-system-ui",
+        "shared-design-system-interfaces"
+      ]
```
Comment (tomheller) [inline nx.json:369]: Did you check the dependency tree if they need to be implicit?

File: libs/shared/data-access-strapi/src/lib/page.service.ts [hunk 2]
```diff
@@ -88,6 +92,11 @@ export class BaPageService<T = any> {
       .get<T>(requestPath, { responseType: 'json' })
       .pipe(tap((data) => this._cache.set(id, data)));
   }
+
+  private _fetchRoutes(): Observable<T> {
+    const requestPath = `data/routes.txt`;
+    return this._http.get<T>(requestPath, { responseType: 'json' });
+  }
```
Comment (tomheller) [inline libs/shared/data-access-strapi/src/lib/page.service.ts:99]: Why do we need this information in there?

Comment (rowa-audil) [reply libs/shared/data-access-strapi/src/lib/page.service.ts:104]: Left over from something I tried. Getting the routes for the top nav but found a better solution I want to implement soon

File: apps/next/src/app/pages/single-page/single-page.html [hunk 3]
```diff
@@ -1,8 +1,10 @@
-<div class="next-page-grid" *ngIf="_pageContent">
+<div class="next-singlepage-grid" *ngIf="_pageContent">
```
Comment (lukasholzer) [inline apps/next/src/app/pages/single-page/single-page.html:1]: NIT:
```suggestion
<div class="next-single-page-grid" *ngIf="_pageContent">
```

File: apps/next/src/app/pages/single-page/single-page.html [hunk 4]
```diff
@@ -1,8 +1,10 @@
...
   <nav next-sidenav *ngIf="_pageContent.sidenav"></nav>
+  <!-- False for now. Until sidenav component is implemented-->
+  <nav></nav>
 
-  <main class="next-page-main" id="main" role="main">
-    <article class="next-single-page-content">
+  <main class="next-singlepage-main" id="main" role="main">
+    <article class="next-singlepage-content">
```
Comment (lukasholzer) [inline apps/next/src/app/pages/single-page/single-page.html:7]: NIT
```suggestion
  <main class="next-single-page-main" id="main" role="main">
    <article class="next-single-page-content">
```

File: apps/next/src/app/pages/single-page/single-page.scss [hunk 5]
```diff
@@ -1,3 +1,9 @@
 :host {
   display: block;
 }
+
+.next-singlepage-grid {
```
Comment (lukasholzer) [inline apps/next/src/app/pages/single-page/single-page.scss:5]: NIT:

```suggestion
.next-single-page-grid {
```

File: apps/next/src/app/pages/single-page/single-page.scss [hunk 6]
```diff
@@ -1,3 +1,9 @@
 :host {
   display: block;
 }
+
+.next-single-page-grid {
+  display: grid;
+  grid-template-columns: 0.3fr 1fr 0.3fr;
```
Comment (tomheller) [inline apps/next/src/app/pages/single-page/single-page.scss:7]: I do prefer whole values here, it is a ratio after all.
```suggestion
  grid-template-columns: 1fr 3fr 1fr;
```

File: apps/next/src/styles.scss [hunk 7]
```diff
@@ -31,12 +32,36 @@ html {
...
+  @include fluid-dt-h4();
+}
+
+h5 {
+  @include fluid-dt-h5();
+}
+
+h6  {
+  @include fluid-dt-h6();
 }
```
[the remark this answers is not in the document]
Comment (rowa-audil) [reply apps/next/src/styles.scss]: We would need a transformer that adds the classes to headlines. I'll create an issue tomorrow

File: apps/next/src/app/pages/single-page/single-page.html [hunk 8]
```diff
@@ -1,7 +1,9 @@
-<div class="next-page-grid" *ngIf="_pageContent">
+<div class="next-single-page-grid" *ngIf="_pageContent">
   <nav next-sidenav *ngIf="_pageContent.sidenav"></nav>
+  <!-- False for now. Until sidenav component is implemented-->
+  <nav></nav>
 
-  <main class="next-page-main" id="main" role="main">
+  <main class="next-single-page-main" id="main" role="main">
```
[the remark this answers is not in the document]
Comment (rowa-audil) [reply apps/next/src/app/pages/single-page/single-page.html:6]: To identify it ;)

File: apps/next/src/styles.scss (hunk 7 above)
[the remark this answers is not in the document]
Comment (rowa-audil) [reply apps/next/src/styles.scss]: In that case we would need a transformer for headline elements coming from strapi

File: apps/next/tsconfig.app.json [hunk 9]
```diff
@@ -2,7 +2,7 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "outDir": "../../dist/out-tsc",
-    "types": []
+    "types": ["css-modules"]
```
Comment (ffriedl89) [inline apps/next/tsconfig.app.json:5]: I guess we don't need this one

File: apps/next/src/styles.scss (hunk 7 above)
[the remark this answers is not in the document]
Comment (rowa-audil) [reply apps/next/src/styles.scss]: Will be resolved in a follow-up issue

File: apps/next/tsconfig.app.json (hunk 9 above)
Comment (rowa-audil) [reply apps/next/tsconfig.app.json:5]: We need this in order for the next app build to succeed

Comment (tomheller) [reply apps/next/tsconfig.app.json:5]: He is correct, we can remov ethis once #1291 is merged, but until then, it is still required.

**Pull request merged**